index.html

```diff
@@ -177,51 +177,50 @@ <h4 class="about">About me</h4>
 
         <section class="form">
             <p class="prg">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-            <form action="">
-                <input
-                class="input"
-                type="text"
-                name="name"
-                id="name"
-                placeholder="Full name"
-                maxlength="30"/>
-            </div>
-            <div class="">
-              <input
-                class="input"
-                type="email"
-                name="email"
-                id="email"
-                placeholder="Email address"
-              />
-            </div>
-            <br>
-            <div class="msg">
-              <textarea
-                name="message"
-                id="message"
-                cols="30"
-                rows="10"
-                placeholder="Enter text here"
-                maxlength="500"     
-              ></textarea>
-              <br>
-            </div>
-            
-              <button class="get-in" type="button">Get It Touch</button>
-              
-                
-            </form>
-            <ul class="list">
-                <li><a href="#"><i class="fa fa-github"></i></a></li>
-                <li><a href="#"><i class="fa fa-linkedin"></i></a></li>
-                <li><a href="#"><i class="fa fa-angellist"></i></a></li>
-                <li><a href="#"><i class="fa fa-twitter"></i></a></li>
-                <li><a href="#"><i class="fa fa-medium"></i></a></li>
-            </ul>
+          
+                <form  action="https://formspree.io/f/xpzbqzab" method="post">
+                    <input
+                    class="input"
+                    type="text"
+                    name="name"
+                    id="name"
+                    placeholder="Full name"
+                    maxlength="30"/>
```

Comment:
- [ ] All input fields should have the `required` attribute to activate the validation on submit. This ensures that a user cannot submit an empty form.